README.md

```diff
@@ -884,8 +884,16 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [SimpleLogin](https://simplelogin.io/) – Open source, self-hostable email alias/forwarding solution. Free 5 Aliases, unlimited bandwith, unlimited reply/send. Free for educational staffs (student, researcher, etc).
   * [Substack](https://substack.com) — Unlimited free newsletter service. Start paying when you charge for it.
   * [Tempmailo](https://tempmailo.com/) - Unlimited free temp email addresses. Autoexpire in two days.
-  * [Takeout](https://takeout.bysourfruit.com) - A constantly updated email service that makes sending emails extremely easy. 500 transactional emails/month free.
+* [Sendpulse](https://sendpulse.com) — 500 subscribers/month, 12,000 emails/month free
+  * [SimpleLogin](https://simplelogin.io/) – Open source, self-hostable email alias/forwarding solution. Free 5 Aliases, unlimited bandwith, unlimited reply/send. Free for educational staffs (student, researcher, etc).
+  * [Substack](https://substack.com) — Unlimited free newsletter service. Start paying when you charge for it.
+  * [Tempmailo](https://tempmailo.com/) - Unlimited free temp email addresses. Autoexpire in two days.
+  * [eTempMail](https://etempmail.net) - Provides multipe temporary email addresses at the same time.
```

Comment:
Please remove, it refuse to work with adblockers on.  Adblockers are essential to online safety.